plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -499,6 +499,25 @@ public MongoCursor<Document> execute(MongoTableHandle tableHandle, List<MongoCol
         return iterable.iterator();
     }
 
+    static Document buildProjection(List<MongoColumnHandle> columns)
+    {
+        Document output = new Document();
+        // _id is always projected by mongodb unless its explicitly excluded.
+        // Explicitly exclude it, if it is not part of the list of columns
+        // https://www.mongodb.com/docs/drivers/java/sync/current/fundamentals/builders/projections/#exclusion-of-_id
+        boolean excludeId = true;
```

Comment:
I think we don't need this flag. We can put `output.append("_id", 0)` before `for` loop.